### ISSUE TITLE
Task provisioning

### ISSUE THREAD

**Issue by cjpatton** [cloudflare/daphne#82]
* Each deployment has a bearer-token authenticated endpoint for configuring tasks:
  * `POST /add_task`: Stores task ID, task config, and bearer token to accept for the task.
     * Validate task config
     * (Optional) Encrypt secrets
  * `POST /delete_task?task_id=<task_id>`: Deletes a task.
 * Any changes should be a proper subset of David's interop test plan.

**Comment by divergentdave** [comment]
The interop test interface defines a JSON object for setting up task parameters in [table 6](https://divergentdave.github.io/draft-dcook-ppm-dap-interop-test-design/draft-dcook-ppm-dap-interop-test-design.html#table-6). If you dropped the "verifyKey" attribute, that would work for these purposes, I think.

The VDAF and its parameters are covered in [table 1](https://divergentdave.github.io/draft-dcook-ppm-dap-interop-test-design/draft-dcook-ppm-dap-interop-test-design.html#table-1), though we plan to change that soon so that "bits" is a string, not a number. (The rationale is that some VDAF-related parameters, measurements or aggregate results cannot safely be transmitted as a number, since they could be 64-bit or 128-bit integers. Making all VDAF-related items strings simplifies reasoning about them and simplifies adding new VDAFs.)

**Comment by cjpatton** [comment]
> (The rationale is that some VDAF-related parameters, measurements or aggregate results cannot safely be transmitted as a number, since they could be 64-bit or 128-bit integers. ...

Why is this not safe? Is the a limitation of JSON?

**Comment by divergentdave** [comment]
Yes, JS's Number type is a 64-bit float, thus JSON implementations commonly use that for internal representation. `serde_json` can use any of a u64, i64, or f64. `jq` will round large numbers beyond 17 digits. I checked the JSON RFC, and it says that large JSON numbers "may indicate potential interoperability problems".

**Comment by cjpatton** [comment]
Thanks David for pointing this out. Notably this is an issue for the VDAF test vectors as well: https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/116
